template/src/pageComponents/buy-me-coffee/steps/BuyCoffeeFormStep/consts.ts

```diff
@@ -0,0 +1,2 @@
+export const BUY_COFFEE_AMOUNT_RAW = 0.0001;
```

Comment:
Side note
buy-me-coffee/steps/BuyCoffeeFormStep/consts.ts
would love if this could be more flat as file structure.

But maybe is a bad idea, I just want to avoid create too many nesting.